tests/FunctionalJavascript/TranslationImporterTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_multilingual\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Tests the translator importer service using a test Controller.
+ */
+class TranslationImporterTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_multilingual',
+    'translation_importer_test',
+  ];
+
+  /**
+   * Tests the translation importer.
+   */
+  public function testTranslationImportBatch(): void {
+    $locale_storage = $this->container->get('locale.storage');
+    $translations = $locale_storage->getTranslations(['source' => 'Structure']);
+    $this->assertEmpty($translations);
+    $this->drupalGet('import-translations');
+    // Wait for the AJAX to complete. This can take a while until the batch
+    // process finishes hence the increase in timeout.
+    $this->assertSession()->assertWaitOnAjaxRequest(100000);
```

Comment:
This can be improved so that we won't wait this much. We will handle it in a follow-up.